src/main/java/seedu/address/logic/commands/ClearWeddingBookCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.model.WeddingBook;
+
+/**
+ * Clears the address book.
+ */
+public class ClearWeddingBookCommand extends Command {
+
+    public static final String COMMAND_WORD = "clear-weddingbook";
```

Comment:
can consider abstracting the clear command into an interface/ abstract class like how we plan to implement add and tag